DependencyInjection/Configuration.php

```diff
@@ -162,13 +172,25 @@ private function getDbalConnectionsNode(): ArrayNodeDefinition
                 ->end()
             ->end();
 
+        // dbal < 2.11
         $slaveNode = $connectionNode
             ->children()
                 ->arrayNode('slaves')
+                    ->setDeprecated(
+                        ...$this->getDeprecationMsg('"slaves" config key will be renamed to "replicas" in doctrine-bundle 3.0. "slaves" is deprecated since doctrine-bundle 2.2.')
```

Comment:
```suggestion
                        ...$this->getDeprecationMsg('The "slaves" configuration key will be renamed to "replicas" in doctrine-bundle 3.0. "slaves" is deprecated since doctrine-bundle 2.2.')
```